.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+branch = True
```

Comment:
Can we just drop `branch` outright? (Or is it's usage strictly required with codecov?)